### ISSUE TITLE
[Feature] Juggler with a given time interval

### ISSUE THREAD

**Issue by b005t3r** [Gamua/Starling-Framework#833]
I was thinking, what if you use a Juggler and sometimes to make sure all tweens added to it finished their execution you call ```juggler.advanceTime(1000000)```. After a couple of such calls you'll end up with Juggler's internal time being much greater than the delta-t passed to ```advanceTime()``` which might lead to lack of precision and tweens to being advanced at all.

I think a solution might be to reset Juggler's internal time once it exceeds a given maximum value (using a simple modulo operation). In most of the cases it's not really relevant what Juggler's internal time is, all it matters if tweens are advanced properly.

Probably adding a ```maxInterval``` property to Juggler would be a good idea.

**Comment by PrimaryFeather** [comment]
Advancing the juggler by `1000000` would mean jumping almost 12 days into the future, so that's probably more than necessary, anyway - but yes, I do see your point! When the value gets high enough, precision will be reduced. Question is by how high the value must grow before we run into problems.

I don't know the exact format the Number format is using internally, but a quick test reveals the following: even when the time reaches the value `1 000 000 000 000` (`10^12`), I still see more than 3 digit precision. That's over 31 thousand years!

In other words, I don't see an immediate call to action here. What do you think?

**Comment by b005t3r** [comment]
I think there might be a values close to a couple of days when an app comes back from the background on mobiles. Other than that only "resetting" (by calling ```advanceTime(1000000)```) a Juggler often enough with a value high enough could cause problems.

**Comment by PrimaryFeather** [comment]
> Other than that only "resetting" (by calling advanceTime(1000000)) a Juggler often enough with a value high enough could cause problems.

Exactly, and even that would need to be done about a million times; and for what this call should achieve, 1000 seconds will probably also work just as well. 

If you agree with me that we can ignore this for now, please just close the issue. :wink:
